content/common/cli-reference.md

```diff
@@ -1238,17 +1314,34 @@ CLI commands have four phases that indicate levels of stability:
 
 {{ startTab "v1.0.0"}}
 
-| Command                                                    | Stability   |
-| ---------------------------------------------------------- | ----------- |
-| <code>spin add</code>                                                 | Stable      |
-| <code>spin build</code>                                               | Stable      |
-| <code>spin new</code>                                                 | Stable      |
+| Command                                                                               | Stability   |
+| ------------------------------------------------------------------------------------- | ----------- |
+| <code>spin add</code>                                                                 | Stable      |
+| <code>spin build</code>                                                               | Stable      |
+| <code>spin new</code>                                                                 | Stable      |
 | <code>spin plugins <install&vert;list&vert;uninstall&vert;update&vert;upgrade></code> | Stable      |
-| <code>spin templates <install&vert;list&vert;uninstall&vert;upgrade></code>       | Stable      |
-| <code>spin up</code>                                                  | Stable      |
-| <code>spin cloud <deploy&vert;login></code>                               | Stabilizing |
-| <code>spin registry</code>                                            | Stabilizing |
-| <code>spin bindle <prepare&vert;push></code>                              | Deprecated  |
+| <code>spin templates <install&vert;list&vert;uninstall&vert;upgrade></code>           | Stable      |
+| <code>spin up</code>                                                                  | Stable      |
+| <code>spin cloud <deploy&vert;login></code>                                           | Stabilizing |
+| <code>spin registry</code>                                                            | Stabilizing |
+| <code>spin bindle <prepare&vert;push></code>                                          | Deprecated  |
+
+{{ blockEnd }}
+
+{{ startTab "v1.1.0"}}
```

Comment:
This might need a "hold off until 1.1" label... though really we need a policy on how we handle new stuff in the docs.  E.g. when we add this to the "running apps" page do we put a "1.1" warning on that section?  How long do we keep those warnings around?  (When we get to Spin 3.8 we don't want to have 20 versions' worth of ancient history.)  Not asking you to solve that, just we need a more general plan, and then we need to retrofit this into that plan.

Reply:
Doh, I see it already has the label!  But the part about a general plan still holds.